backend/tests/contract/conftest.py

```
@@ -0,0 +1,57 @@
+
+
+# Load the committed snapshot from disk (source of truth per PRD #262).
+# Then attach the FastAPI app so .transport auto-resolves to ASGITransport
+# (Schemathesis 4.x: schema.transport is derived from schema.app).
+schema = schemathesis.openapi.from_path(str(SPEC_PATH))
+schema.app = app
+
+
+# Hypothesis tuning: cut max_examples drastically. We're testing "does the
+# response shape match the spec?" not "find a Hypothesis-generated edge
+# case". 5 examples per endpoint × ~63 endpoints = ~315 calls; should stay
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)